.gitignore

```diff
@@ -20,15 +20,12 @@ data/*
 
 # these are our 'dist' files
 # checking them into the repo is silly
-webcompat/static/js/build/
-webcompat/static/js/webcompat.js
-webcompat/static/js/diagnose.js
-webcompat/static/js/issues.js
-webcompat/static/js/issue-list.js
-webcompat/static/js/user-activity.js
-webcompat/static/js/templates.js
 webcompat/**/*.min.js
-webcompat/**/**/*.min.css
+webcompat/static/js/*.js
```

Comment:
Do we need this line? In theory there shouldn't be anything there, right?

(or is it just to ignore the already existing files so people don't have to manually delete them?)

Reply:
Yes, it's to make sure, that there are no accidental files or leftovers from older project states.